Cargo.toml

```diff
@@ -32,13 +34,14 @@ rand = "0.8.5"
 regex = "1.7.1"
 serde = { version = "1.0.155", features = ["derive"] }
 bitvec = { version = "1.0.1", features = ["serde"] }
-serde_json = { version = "1.0.94", features = ["raw_value"]}
+serde_json = { version = "1.0.94", features = ["raw_value"] }
 sha2 = "0.10.6"
 tempfile = "3.4.0"
 thiserror = "1.0.39"
-ursa = { version = "0.3.7", default-features = false, features = ["cl_native", "serde"] }
 zeroize = { version = "1.5.7", optional = true, features = ["zeroize_derive"] }
+anoncreds-clsignatures = "0.1"
 
+# TODO: Can we remove this since ursa is no longer a dependency?
```

Comment:
`anoncreds-clsignatures` has an `openssl_vendored` feature that you can use instead.